src/Path/Name.hs

```diff
@@ -129,28 +129,32 @@ instance Pretty Qualified where
   pretty (m :.: n) = pretty m <> dot <> pretty n
 
 
-data Name
+data Name a
```

Comment:
Parameterizing `Name` by the type of locals has allowed us to change the factoring s.t. `>>=` on `Core`/`Value` never operates on globals. This in turn has resolved some errors in the solver, and I don’t understand how 😭